home-application/src/main/resources/config/application.properties

```diff
@@ -1,4 +1,3 @@
-#spring.profiles.active=dev
 management.server.port=${MANAGEMENT_SERVER_PORT:8090}
 management.endpoints.web.exposure.include=health
 management.endpoint.health.show-details=always
```

Comment:
Please add this property:
`spring.jpa.hibernate.ddl-auto=validate`

Reply:
Done.